mixins/loading-complete/loading-complete-mixin.js

```diff
@@ -22,7 +22,18 @@ export const LoadingCompleteMixin = dedupeMixin((superclass) => class extends su
 	#loadingCompleteResolve;
 
 	#loadingCompletePromise = !Object.prototype.hasOwnProperty.call(this.constructor.prototype, 'getLoadingComplete')
-		? new Promise(resolve => this.#loadingCompleteResolve = resolve)
+		? new Promise(resolve => {
+			const TIMEOUT_MS = 1000 * 10;
```

Comment:
I guess this could be longer (i.e. 30 secs? 1 min?) in case we happen to be working with very slow components

Reply:
Super nit: calculating the ms seems sorta unnecessary. I'd be inclined to just write `const timeout = 30000;`. Also, while we _sometimes_ use uppercase formatting for module scoped constants, we don't throughout the code.

Reply:
I would just bump `TIMEOUT_MS` out to the top-level module scope since it's static